packages/yum/install_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -exu
+
+package="$1"
+
+os_version=$(cut -d: -f5 /etc/system-release-cpe)
+mariadb_version=$(echo "${package}" | cut -d'-' -f2)
+
+cat > /etc/yum.repos.d/MariaDB.repo <<EOF
+[mariadb]
+name = MariaDB
+baseurl = "http://yum.mariadb.org/${mariadb_version}/centos${os_version}-amd64"
```

Comment:
このURLってどこから持ってきた？

https://mariadb.org/download/?t=repo-config&d=Red+Hat+Enterprise+Linux+8&v=10.5&r_m=yamagata-university だと違うURLをオススメされるよ。

```suggestion
baseurl = "https://rpm.mariadb.org/${mariadb_version}/rhel/\$releasever/\$basearch"
```

Reply:
Mroongaの公式ドキュメントから持ってきました。。。
https://mroonga.org/ja/docs/install/almalinux.html#almalinux-8-with-mariadb-10-5-package

MariaDBの方でオススメされているのを使うようにします。
ドキュメントの方も直します。